spark-jobs/src/main/scala/za/co/absa/enceladus/common/CommonJobExecution.scala

```diff
@@ -83,14 +86,17 @@ trait CommonJobExecution {
     dao.authenticate()
     val dataset = dao.getDataset(cmd.datasetName, cmd.datasetVersion)
     val reportVersion = getReportVersion(cmd, dataset)
-    val pathCfg = getPathCfg(cmd, dataset, reportVersion)
+    val pathCfg = getPathConfig(cmd, dataset, reportVersion)
+
+    val (inputPath, outputPath) = getInputOutputPaths(cmd, fsUtils, pathCfg)
+
+    log.info(s"input path: $inputPath")
+    log.info(s"output path: $outputPath")
```

Comment:
Would move this logging to `prepareStandardization`/`prepareConformace`, and make it specific "Standardization input path: ..." etc,